repository_service_tuf_worker/services/storage/local.py

```diff
@@ -62,9 +60,9 @@ def get(self, role, version=None) -> BufferedReader:
         file_object = None
         try:
             file_object = open(filename, "rb")
-            yield file_object
-        except OSError:
-            raise StorageError(f"Can't open Role '{role}'")
+            return Metadata.from_bytes(file_object)
```

Comment:
It fails once it doesn't read the `file_object`:

```log
repository-service-tuf-worker_1  | Traceback (most recent call last):
repository-service-tuf-worker_1  |   File "/usr/local/lib/python3.10/site-packages/tuf/api/serialization/json.py", line 36, in deserialize
repository-service-tuf-worker_1  |     json_dict = json.loads(raw_data.decode("utf-8"))
repository-service-tuf-worker_1  | AttributeError: '_io.BufferedReader' object has no attribute 'decode'
```

```suggestion
            return Metadata.from_bytes(file_object.read())
```

Reply:
Thanks for noticing that!
I will try to test my prs better so you don't have to notice these things.